spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/actions/DeleteReachableFilesSparkAction.java

```diff
@@ -132,7 +135,20 @@ private Dataset<FileInfo> reachableFileDS(TableMetadata metadata) {
   }
 
   private DeleteReachableFiles.Result deleteFiles(Iterator<FileInfo> files) {
-    DeleteSummary summary = deleteFiles(deleteExecutorService, deleteFunc, files);
+    DeleteSummary summary;
+    if (deleteFunc != null || !(io instanceof SupportsBulkOperations)) {
+      if (deleteFunc == null) {
+        LOG.info("Table IO does not support Bulk Operations. Using non-bulk deletes.");
```

Comment:
nit: Why capital letter for some words?

Reply:
changed this and made it so the same messages appear in all three actions